packages/components/src/color-palette/style.scss

```
@@ -26,10 +26,10 @@ $border-as-box-shadow: inset 0 0 0 $border-width rgba(0, 0, 0, 0.2);
 	&::after {
 		content: "";
 		position: absolute;
-		top: 0;
-		left: 0;
-		width: 100%;
-		height: 100%;
+		top: 1px;
+		left: 1px;
+		right: 1px;
+		bottom: 1px;
```

Should these use `$border-width`?

```suggestion
		top: $border-width;
		left: $border-width;
		right: $border-width;
		bottom: $border-width;
```

---

What about

```suggestion
		inset: $border-width;
```

as a shorthand?

---

Even better 👍